webserver/main/utils/validation.py

```diff
@@ -31,19 +31,6 @@ def validate_payload_schema_based_on_version(request_payload, request_type):
                                    "message": "Version should be 1.2.0 !"}), 400
 
 
-def validate_payload_schema_using_json_schema(request_payload, request_type):
```

Comment:
reason of removing it ? are we using it anywhere

Reply:
validate_payload_schema_based_on_version is not using validate_payload_schema_using_json_schema as 1.2.0 is using validate_payload_schema_using_pydantic_classes or sending response error 400.
I can revert this back if you want.

Reply:
ok i can see that this is not being used anywhere, lets go ahead no worries